database/pgsql/migrations/00001_initial_schema.go

```diff
@@ -14,179 +14,220 @@
 
 package migrations
 
-import "github.com/remind101/migrate"
+import "github.com/coreos/clair/pkg/dbutil"
 
-func init() {
-	RegisterMigration(migrate.Migration{
-		ID: 1,
-		Up: migrate.Queries([]string{
+var (
```

Comment:
Schema migration queries are now separated from each other to have a cleaner structure.